src/libraries/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/SymmetricAlgorithm.cs

```diff
@@ -222,6 +222,209 @@ public bool ValidKeySize(int bitLength)
+
+            switch (paddingMode)
+            {
+                case PaddingMode.None when remainder != 0:
+                    throw new ArgumentException(SR.Cryptography_MatchBlockSize, nameof(plaintextLength));
+                case PaddingMode.None:
+                case PaddingMode.Zeros when remainder == 0:
+                    return plaintextLength;
+                case PaddingMode.Zeros:
+                case PaddingMode.PKCS7:
+                case PaddingMode.ANSIX923:
+                case PaddingMode.ISO10126:
+                    if (int.MaxValue - wholeBlocks < blockSizeBytes)
+                    {
+                        throw new ArgumentOutOfRangeException(nameof(plaintextLength), SR.Cryptography_PlaintextTooLarge);
+                    }
+
+                    return wholeBlocks + blockSizeBytes;
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(paddingMode), SR.Cryptography_InvalidPaddingMode);
+            }
+        }
+
+
+        /// <summary>
+        /// Gets the length of a ciphertext with a given padding mode and plaintext length in CFB mode.
+        /// </summary>
+        /// <param name="paddingMode">The padding mode used to pad the plaintext to the feedback size.</param>
+        /// <param name="plaintextLength">The plaintext length, in bytes.</param>
+        /// <param name="feedbackSizeInBits">The feedback size, in bits.</param>
+        /// <returns>The length, in bytes, of the ciphertext with padding.</returns>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///   <para>
+        ///   <paramref name="feedbackSizeInBits" /> is not a positive number.
+        ///   </para>
+        ///   <para>
+        ///   - or -
+        ///   </para>
+        ///   <para>
+        ///   <paramref name="plaintextLength" /> is a negative number.
+        ///   </para>
+        ///   <para>
+        ///   - or -
+        ///   </para>
+        ///   <para>
+        ///   <paramref name="plaintextLength" /> when padded is too large to represent as
+        ///   a signed 32-bit integer.
+        ///   </para>
+        ///   <para>
+        ///   - or -
+        ///   </para>
+        ///   <para>
+        ///   <paramref name="paddingMode" /> is not a valid padding mode.
+        ///   </para>
+        /// </exception>
+        /// <exception cref="ArgumentException">
+        ///   <para>
+        ///   The padding mode <see cref="PaddingMode.None" /> was used, but <paramref name="plaintextLength" />
+        ///   is not a whole number of blocks.
+        ///   </para>
+        ///   <para>
+        ///   - or -
+        ///   </para>
+        ///   <para>
+        ///   <paramref name="feedbackSizeInBits" /> is not a whole number of bytes. It must be divisible by 8.
+        ///   </para>
+        /// </exception>
+        public int GetCiphertextLengthCfb(int plaintextLength, PaddingMode paddingMode = PaddingMode.None, int feedbackSizeInBits = 8)
```

Comment:
Is it worth a `<remarks>` that this will produce an output even if the algorithm/implementation doesn't support a particular specified feedback size?  (e.g. none of our types support CFB16, but this'll produce an answer for that)